packages/common/src/lib/format-appeal-details.js

```diff
@@ -1,29 +1,73 @@
 /**
- * @param {import('appeals-service-api').Api.Document[]} documents
- * @param {string} documentType
+ * @param {import("../client/appeals-api-client").AppealCaseWithAppellant} caseData
  */
-exports.formatDocumentDetails = (documents, documentType) => {
-	const filteredDocuments = documents.filter((document) => document.documentType === documentType);
+exports.formatContactDetails = (caseData) => {
+	const contactName = `${caseData.contactFirstName} ${caseData.contactLastName}`;
 
-	return filteredDocuments.length > 0 ? filteredDocuments.map(formatDocumentLink).join('\n') : 'No';
+	return contactName + (caseData.contactCompanyName ? `\n${caseData.contactCompanyName}` : '');
 };
 
 /**
  * @param {import("../client/appeals-api-client").AppealCaseWithAppellant} caseData
  */
-exports.formatNewDescription = (caseData) => {
-	if (caseData.updateDevelopmentDescription && caseData.developmentDescriptionDetails) {
-		return caseData.developmentDescriptionDetails;
-	}
+exports.formatApplicantDetails = (caseData) => {
+	const contactName = `${caseData.appellantFirstName} ${caseData.appellantLastName}`;
 
-	return 'No';
+	return contactName + (caseData.appellantCompanyName ? `\n${caseData.appellantCompanyName}` : '');
 };
 
 /**
- *
- * @param {import('appeals-service-api').Api.Document} document
- * @returns {string}
+ * @param {import("../client/appeals-api-client").AppealCaseWithAppellant} caseData
  */
-const formatDocumentLink = (document) => {
-	return `<a href=# class="govuk-link">${document.filename}</a>`;
+exports.formatAccessDetails = (caseData) => {
+	const visibility = caseData.appellantSiteAccess ? 'Yes' : 'No';
+
+	return (
+		visibility +
+		(caseData.appellantSiteAccessDetails ? `\n${caseData.appellantSiteAccessDetails}` : '')
+	);
+};
+
+/**
+ * @param {import("../client/appeals-api-client").AppealCaseWithAppellant} caseData
+ */
+exports.formatHealthAndSafety = (caseData) => {
+	const safetyIssues = caseData.appellantSiteSafety ? 'Yes' : 'No';
+
+	return (
+		safetyIssues +
+		(caseData.appellantSiteSafetyDetails ? `\n${caseData.appellantSiteSafetyDetails}` : '')
+	);
+};
+
+/**
+ * @param {import("../client/appeals-api-client").AppealCaseWithAppellant} caseData
+ */
+exports.formatProcedure = (caseData) => {
+	const possibleProcedures = [
+		caseData.appellantProcedurePreference ?? '',
+		caseData.appellantPreferHearingDetails ?? '',
+		caseData.appellantPreferInquiryDetails ?? ''
+	];
+
+	const valueText = possibleProcedures.filter(Boolean).join('\n');
+
+	return valueText;
+};
+
+/**
+ * @param {import("../client/appeals-api-client").AppealCaseWithAppellant} caseData
+ */
+exports.formatLinkedAppeals = (caseData) => {
+	if (!caseData.appellantLinkedCase) return 'No';
+
+	const appellantLinkedCases = caseData.SubmissionLinkedCase || [];
+
+	const linkedCasesDisplayDetails = appellantLinkedCases
+		.map((linkedCase) => linkedCase.caseReference)
+		.join('\n');
```

Comment:
I think Steve said these would be links to the appeals, but I'm sure that's covered in a ticket somewhere

Reply:
I've changed these to be links but left the href blank for now.  A lot of the links for these dashbaordas / pages are dummy links at present...